.gitignore

```diff
@@ -9,3 +9,5 @@ dist
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.prettierrc
```

Comment:
What was the motivation add this?

Reply:
I can remove it. Was for the file not to be commited to git

Reply:
I recommend you to use a [global gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/), to avoid commit your env specific files

Reply:
Thanks for the suggestion, but as prettier settings are project specific I usually commit them to git.